From 39f2630961a5c70fd4dc09870d5fd7fa7b4781b6 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 23 Jun 2014 20:01:10 -0400
Subject: [scsi] qla2xxx: Clear loop_id for ports that are marked lost during fabric scanning

Message-id: <1403553672-3844-5-git-send-email-cdupuis@redhat.com>
Patchwork-id: 84414
O-Subject: [PATCH RHEL 7.1 e-stor 4/6] qla2xxx: Clear loop_id for ports that are marked lost during fabric scanning.
Bugzilla: 1090378
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

From: Chad Dupuis <chad.dupuis@qlogic.com>

Bug 1090378 (https://bugzilla.redhat.com/show_bug.cgi?id=1090378)

scsi commit id 1a5c69bf0c3c157ed438cca4b218aca455b30019

Signed-off-by: Chad Dupuis <chad.dupuis@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_init.c b/drivers/scsi/qla2xxx/qla_init.c
index 03f715e..2802b7b 100644
--- a/drivers/scsi/qla2xxx/qla_init.c
+++ b/drivers/scsi/qla2xxx/qla_init.c
@@ -3276,7 +3276,7 @@ qla2x00_configure_fabric(scsi_qla_host_t *vha)
          fcport->d_id.b.domain,
          fcport->d_id.b.area,
          fcport->d_id.b.al_pa);
-     fcport->loop_id = FC_NO_LOOP_ID;
+     qla2x00_clear_loop_id(fcport);
     }
    }
   }
-- 
1.7.1